builder/dockerfile/builder.go

```diff
@@ -87,12 +89,13 @@ type Builder struct {
 
 // BuildManager implements builder.Backend and is shared across all Builder objects.
 type BuildManager struct {
-	backend builder.Backend
+	backend   builder.Backend
+	pathCache *pathCache // TODO: make this persistent
```

Comment:
It is for the future. Currently, the cache checksum needs to be recomputed after daemon restarts.